components/compas-open/test/compas-open.test.ts

```diff
@@ -0,0 +1,35 @@
+import { fixture } from "@open-wc/testing";
+import { html } from "lit";
+import { visualDiff } from "@web/test-runner-visual-regression";
+
+import "../src/CompasOpen.js";
+import { CompasOpenElement } from "../src/CompasOpen.js";
```

Comment:
import type { CompasOpenElement }

otherwise sonarcloud will fail because you're importing twice from src/CompasOpen.js